fud/fud/main.py

```diff
@@ -421,6 +424,7 @@ def config_run(parser):
     )
     parser.add_argument("-q", "--quiet", action="store_true")
     parser.add_argument("input_file", help="Path to the input file", nargs="?")
+    parser.add_argument("-i", "--include", dest="include")
```

Comment:
It looks like this solution has added a new top-level CLI argument to fud. That's aright, but it seems like it might be a little less disruptive (and more useful) to add a configuration option to the Icarus and Verilator stages specifically? That is, fud already has a mechanism for config options that are stage-specific, so you can put them in your config file and forget about them.

For example, this is where we make the Verilator executable name configurable:
https://github.com/calyxir/calyx/blob/ce57c8e4652be1f90a88dd26fdb2b3d2ee8430b1/fud/fud/stages/verilator/stage.py#L164

So the user can put `verilator.exec = "whatever"` in their config file. Maybe we should do that for the include path too, which would involve a bit less plumbing than the CLI option and also allow persistent configuration.

Reply:
Agree! Got rid of it!